src/locales/pt.json

```
@@ -0,0 +1,217 @@
+{
+  "Blockchain": {
+    "Mnemonic": "Mnemônico",
+    "Address": "Endereço",
+    "Wallet": "Carteira",
+    "Wallets": "Carteiras",
+    "Coins": "Moedas",
+    "Tokens": "Tokens",
+    "Balance": "Saldo",
+    "Tx hash": "Hash Tx",
+    "Gas": "Gas",
+    "Tax": "Imposto",
+    "Fee": "Taxa",
+    "Memo": "Memo",
+    "decimals": "decimais",
+    "Buy": "Comprar",
+    "Exchanges": "Exchanges",
+    "Fiat": "Fiat",
+    "APY": "APY"
+  },
+  "Form": {
+    "Amount": "Quantidade",
+    "Recipient": "Destinatário",
+    "Name": "Nome",
+    "optional": "opcional",
+    "Submit": "Enviar",
+    "Confirm": "Confirmar",
+    "Reset": "Reset"
+  },
+  "Validate": {
+    "{{label}} is required": "{{label}} é obrigatório",
+    "{{label}} is not defined": "{{label}} não foi definido",
+    "{{label}} is not available": "{{label}} não está disponível",
+    "{{label}} is not supported": "{{label}} não é suportado",
+    "{{label}} is invalid": "{{label}} não é válido",
+    "{{label}} already exists": "{{label}} já existe",
+    "{{label}} does not exist": "{{label}} não existe",
+    "{{label}} must be longer than {{length}} characters": "{{label}} deve ter mais de {{length}} caracteres",
+    "Already exists": "Já existe",
+    "Invalid": "Inválido",
+    "Invalid {{label}}": "{{label}} inválido"
+  },
+  "Loading": {
+    "Initializing app...": "Inicializando a aplicação...",
+    "Loading...": "Carregando...",
+    "Load more": "Carregar mais",
+    "Loading more...": "Carregando mais...",
+    "Nothing more to load": "Não há nada para carregar",
```

Just a 1:1 translation, but of course, both will work :) 
```suggestion
    "Nothing more to load": "Nada mais para carregar",
```

---

Then I don't know. To me it sounds correct Nada mais _para_ carregar, but then if that is wrong, that is wrong :)

---

Your version looks better, I just did a small change because in this case I think we should use "a" instead of "para". I think it is more clear why we should use the article "a" if we look at the phrase "nada a temer", "nada para temer" sounds really strange.

---

Your version looks better, I fixed it.